.ruby-version

```diff
@@ -1 +1 @@
-2.6.2
+2.6.6
```

Comment:
```suggestion
2.6
```

and then it shouldn't need bumped again for a while 😀

Reply:
Fine, there you go!